server/lib/genome/importers/file_importers/drugbank.rb

```diff
@@ -0,0 +1,246 @@
+module Genome; module Importers; module FileImporters; module Drugbank;
+  class Importer < Genome::Importers::Base
+    attr_reader :file_path
+    attr_reader :source_db_name
+    attr_reader :source
+    attr_reader :drug_filter
+
+    def initialize(file_path)
+      @file_path = file_path
+      @source_db_name = 'DrugBank'
+    end
+
+    def run_parser
+      @drug_filter = DrugFilter.new
+      parser = Nokogiri::XML::SAX::Parser.new(drug_filter)
+      parser.parse(File.open(file_path))
+    end
+
+    def create_new_source
+      @source ||= Source.create(
+        {
+            base_url: 'https://go.drugbank.com/',
+            site_url: 'https://go.drugbank.com/',
+            citation: "DrugBank 5.0: a major update to the DrugBank database for 2018. Wishart DS, Feunang YD, Guo AC, Lo EJ, Marcu A, Grant JR, Sajed T, Johnson D, Li C, Sayeeda Z, Assempour N, Iynkkaran I, Liu Y, Maciejewski A, Gale N, Wilson A, Chin L, Cummings R, Le D, Pon A, Knox C, Wilson M. Nucleic Acids Res. 2017 Nov 8. doi 10.1093/nar/gkx1037. PubMed ID: 29126136",
+            source_db_version: '5.1.9',
+            source_db_name: 'DrugBank',
+            full_name: 'DrugBank - Open Data Drug & Drug Target Database',
+            license: '',
```

Comment:
Probably enough to just say "custom". Some other importers provide more of a description if you feel up for it.